src/Api/Soap/Soap.php

```diff
@@ -78,17 +84,16 @@ class Soap extends ApiBase  implements DispatcherAwareInterface
 	 * Method to instantiate the file-based api call.
 	 *
 	 * @param   Container  $container  The DIC object
+	 * @param   Renderer   $renderer   Renderer object.
 	 * @param   Registry   $options    Optional custom options to load
 	 *
 	 * @throws  \Exception
 	 * @since   1.4
 	 */
-	public function __construct(Container $container, Registry $options)
+	public function __construct(Container $container, $renderer, Registry $options)
```

Comment:
Why are we injecting the renderer object here? it seems to be unused and none of the other classes extending ApiBase nor ApiBase itself have this parameter